libs/payments/cart/src/gql/tax-address.model.ts

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType({
+  description: 'The Tax Address associated with the Cart',
+})
+export class TaxAddress {
+  @Field({ nullable: true, description: 'Error reason ID.' })
+  public countryCode!: string;
+
+  @Field({ description: 'Offering ID configured in the CMS.' })
```

Comment:
s/Offering ID configured in the CMS/Postal code for tax/

Reply:
Thank you! 🙏